pkg/sandbox/config.go

```diff
@@ -0,0 +1,35 @@
+package sandbox
+
+import "github.com/flyteorg/flytectl/pkg/docker"
+
+//Config holds configuration flags for sandbox command.
+type Config struct {
```

Comment:
@pmahindrakar-oss my motivation for moving this struct here is that I made this part of the inputs to some of the functions in `pgk/sandbox`. As shown in the signatures below

```golang
func StartCluster(ctx context.Context, args []string, sandboxConfig *Config, primePod bool, defaultImageName string)
func StartDemoCluster(ctx context.Context, args []string, sandboxConfig *Config)
```
what do you think it is better?
- just pass all of this fields directly into these functions
- define this config in config/subcommand and then import it in pkg/sanbox?  ( I wonder if this might cause an import loop)
- create another struct essentially with the same fields. Maybe we can embed them?
```golang
// lives in cmd/config/subcommand
type CLIConfig struct {
   // sandbox.Config
   Config
}
```

Reply:
Option 2 & 3 should be good. 
For the option 2 . I dont see an import loop issue since 

subcommand is only imported in cmd module
pkg is imported in cmd module
here you are introducing a dependency from subcommand -> pkg which should be fine.

Option 3 might lead to issue with pflag generation though as embedded struct may not be supported but you can check.